ips/services/new_run_steps/step_5.py

```diff
@@ -1,81 +1,104 @@
 from flask import request, render_template, session, jsonify
 from ips.util.ui_logging import log
-
+from flask_wtf.file import FileRequired
 
 from ips.services import app_methods
 from ips.services.forms import LoadDataForm
 
-
 def run_step_5(run_id):
-    form = LoadDataForm()
+    from remote_pdb import RemotePdb
+    #RemotePdb('0.0.0.0', 4445).set_trace() 
```

Comment:
You still seem to have lots of commented out pdb's in your code and imports of remote_pdb, looking for matches with ag inside of the ips_user_interface directory I get:

```
$ ag --python -i pdb
ips/services/new_run_steps/step_5.py
9:    from remote_pdb import RemotePdb
10:    #RemotePdb('0.0.0.0', 4445).set_trace() 
14:    #RemotePdb('0.0.0.0', 4445).set_trace() 
27:    #RemotePdb('0.0.0.0', 4445).set_trace() 

tests/test_step_5.py
23:from remote_pdb import RemotePdb
113:        #RemotePdb('0.0.0.0', 4445).set_trace()
```